<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="album_treeView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_AddAlbum"/>
    <addaction name="separator"/>
    <addaction name="action_Login"/>
    <addaction name="action_Logout"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="action_Pause"/>
    <addaction name="separator"/>
    <addaction name="action_Relocate"/>
    <addaction name="action_Delete"/>
    <addaction name="separator"/>
    <addaction name="action_DecreasePriority"/>
    <addaction name="action_IncreasePriority"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Settings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_AddAlbum"/>
   <addaction name="action_Delete"/>
   <addaction name="separator"/>
   <addaction name="action_Start"/>
   <addaction name="action_Pause"/>
   <addaction name="separator"/>
   <addaction name="action_DecreasePriority"/>
   <addaction name="action_IncreasePriority"/>
  </widget>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Login">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/login.png</normaloff>:/icons/login.png</iconset>
   </property>
   <property name="text">
    <string>Log &amp;in...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Logout">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/logout.png</normaloff>:/icons/logout.png</iconset>
   </property>
   <property name="text">
    <string>Log &amp;out...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_AddAlbum">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Album...</string>
   </property>
   <property name="toolTip">
    <string>Add Album</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/qt.png</normaloff>:/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/restart.png</normaloff>:/icons/restart.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="toolTip">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/pause.png</normaloff>:/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="action_DecreasePriority">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/prioritydec.png</normaloff>:/icons/prioritydec.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Decrease priority</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_IncreasePriority">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/priorityinc.png</normaloff>:/icons/priorityinc.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Increase priority</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="action_Relocate">
   <property name="text">
    <string>&amp;Relocate...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
